amplicon-pooling.ipynb

```diff
@@ -0,0 +1,1063 @@
+       "      <td>2021-08-17</td>\n",
+       "      <td>978215</td>\n",
+       "      <td>RNBJ0628</td>\n",
+       "      <td>Echo550</td>\n",
+       "      <td>THDMI_UK_Plate_4</td>\n",
+       "      <td>THDMI UK</td>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>4</td>\n",
+       "      <td>4</td>\n",
+       "      <td>AS</td>\n",
+       "      <td>166032128</td>\n",
+       "      <td>Carmen_HOWE_KF4</td>\n",
+       "      <td>109379Z</td>\n",
+       "      <td>2021-08-17</td>\n",
+       "      <td>978215</td>\n",
+       "      <td>RNBJ0628</td>\n",
+       "      <td>Echo550</td>\n",
+       "      <td>THDMI_US_Plate_6</td>\n",
+       "      <td>THDMI US</td>\n",
+       "      <td></td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "  Plate Position Primer Plate # Plating Extraction Kit Lot Extraction Robot  \\\n",
+       "0              1              1      SF          166032128  Carmen_HOWE_KF3   \n",
+       "1              2              2      AS          166032128  Carmen_HOWE_KF4   \n",
+       "2              3              3   MB_SF          166032128  Carmen_HOWE_KF3   \n",
+       "3              4              4      AS          166032128  Carmen_HOWE_KF4   \n",
+       "\n",
+       "  TM1000 8 Tool Primer Date MasterMix Lot Water Lot Processing Robot  \\\n",
+       "0       109379Z  2021-08-17        978215  RNBJ0628          Echo550   \n",
+       "1       109379Z  2021-08-17        978215  RNBJ0628          Echo550   \n",
+       "2       109379Z  2021-08-17        978215  RNBJ0628          Echo550   \n",
+       "3       109379Z  2021-08-17        978215  RNBJ0628          Echo550   \n",
+       "\n",
+       "       Sample Plate Project_Name Original Name  \n",
+       "0  THDMI_UK_Plate_2     THDMI UK                \n",
+       "1  THDMI_UK_Plate_3     THDMI UK                \n",
+       "2  THDMI_UK_Plate_4     THDMI UK                \n",
+       "3  THDMI_US_Plate_6     THDMI US                "
+      ]
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "_metadata = [\n",
+    "    {\n",
+    "        # top left plate\n",
+    "        'Plate Position': '1',\n",
+    "        'Primer Plate #': '1',\n",
+    "\n",
+    "        'Plating': 'SF',\n",
+    "        'Extraction Kit Lot': '166032128',\n",
+    "        'Extraction Robot': 'Carmen_HOWE_KF3',\n",
+    "        'TM1000 8 Tool': '109379Z',\n",
+    "        'Primer Date': '2021-08-17', # yyyy-mm-dd\n",
+    "        'MasterMix Lot': '978215',\n",
+    "        'Water Lot': 'RNBJ0628',\n",
+    "        'Processing Robot': 'Echo550',\n",
+    "        'Sample Plate': 'THDMI_UK_Plate_2',\n",
+    "        'Project_Name': 'THDMI UK',\n",
+    "        'Original Name': ''\n",
```

Comment:
All these have the same thing than in the test: `'Original Name': '',`; if is always empty, is it needed?

Reply:
I'll defer this for now while we get some feedback from Gail.